F1/F1/Race/Repository/RaceRepository.swift

```diff
@@ -19,7 +19,19 @@ protocol RaceRepositoryType: AnyObject {
 class RaceRepository: RaceRepositoryType {
 
     func fetchRaceResults(completion: @escaping RaceResults) {
-        let url = Endpoints.racing
-        URLSession.shared.request(endpoint: url, method: .GET, completion: completion)
```

Comment:
just checking you intentionally want to remove the fetching from the endpoint and fetch from core data?
Else create 2 different functions f you want both

Reply:
Yes this is intentional. So with this feature it is the race calendar. The calendar is set a year in advance and can not change, so there would be no new data to fetch. For the other features I am doing it differently.